packages/xchain-binance/src/client.ts

```diff
@@ -394,13 +370,21 @@ class Client implements BinanceClient, XChainClient {
    * @param {MultiSendParams} params The multi-send transfer options.
    * @returns {TxHash} The transaction hash.
    */
-  multiSend = async ({ address, transactions, memo = '' }: MultiSendParams): Promise<TxHash> => {
+  multiSend = async ({ address = 0, transactions, memo = '' }: MultiSendParams): Promise<TxHash> => {
```

Comment:
Same here: Try to introduce `index?: number` here (`0` by default)

Reply:
agree